src/lib/utils/hooks/SynapseAPI/user/useFavorites.ts

```diff
@@ -13,6 +16,57 @@ import {
   FavoriteSortDirection,
 } from '../../../synapseTypes/FavoriteSortBy'
 
+const FAVORITES_QUERY_KEY = 'favorites'
+
+export function useIsFavorite(entityId: string) {
+  // TODO: Handle pagination - the default limit is 200
+  // It would probably make more sense to add a backend service to check if an entity ID is favorited
```

Comment:
This is the same as the current SWC implementation, so not introducing any new issues here.